.github/workflows/build.yml

```diff
@@ -24,24 +24,30 @@ jobs:
           - "6.0"
           - "6.1"
           - main
+        include:
+          - ruby: "3.1"
+            rails: "7.0"
+          - ruby: "3.1"
+            rails: "6.1"
         exclude:
           - ruby: 2.5
             rails: main
           - ruby: 2.6
             rails: main
           - ruby: jruby-9.2.16.0
             rails: main
-          - ruby: 3.0
+          - ruby: "3.0"
```

Comment:
I'm not sure I understand why we need quotes here but don't for Ruby 2.5 and 2.6

Reply:
👋  As @petergoldstein mentioned in the PR description it is using 3.1 otherwise.
I believe it is because of this https://github.com/actions/runner/issues/849
It is also mentioned here https://github.com/ruby/setup-ruby#matrix-of-ruby-versions

Reply:
@berkos Yep, that's it.

Reply:
😭 Thank you for the explanation. I saw that in the PR description, but didn't understand why.